src/state/store.ts

```diff
@@ -39,21 +44,27 @@ const handleAbortMiddleware = store => next => action => {
   next(action);
 };
 
-const createStore = intialState =>
+const reducer = combineReducers({
+  account: accountReducer,
+  userHome: userHomeReducer,
+  group: groupReducer,
+  landscape: landscapeReducer,
+  notifications: notificationsReducer,
+  sharedData: sharedDataReducer,
+  taxonomies: taxonomiesReducer,
+  gis: gisReducer,
+  storyMap: storyMapReducer,
+});
+
+type GetReducerType<T extends Reducer> = T extends Reducer<infer S> ? S : never;
```

Comment:
comment: Cool! didn't know this could be done. Is there any cases where `GetReducerType<sometype>` would evaluate to `never`?

Reply:
no, since you have a constraint that T must extend reducer on the lefthand side. IMO the type-level language is weird and gross and should be avoided at all costs, but i don't think there's any choice here.